README.md

```diff
@@ -126,7 +126,7 @@ The `"url"` field jupyter-fs config is based on the PyFilesystem [opener url](ht
 If you prefer to set up your filesystem resources in the server-side config, you can do so. For example, you can set up a local filesystem by adding the following to your `jupyter_notebook_config.py` file:
 
 ```python
-c.jupyterfs.resources = [
+c.Jupyterfs.resources = [
```

Comment:
we should probably make a note to bump to 0.1.0

Reply:
sure, if you think that will make it clearer that a breaking change has occurred. If we go that way, I think it would be good to bump the version to `0.20.0`, to indicate that this version is the Jlab 2.x compatible one. That's the same convention that `jupyterlab-git` follows (ie while the extension is still in development, the `0.1x.y` versions are jlab 1.x compatible, the `0.2x.y` versions are jlab 2.x compatible, etc).

If we don't follow some convention like this, things will likely get confusing once we start working on a jlab 3.x compatible branch

Reply:
0.2.* vs 0.3.* would be sufficient, I don't think theres any need to support jlab 2 once we port to jlab 3 so it won't be too confusing to manage

Reply:
We're still going to need `0.3a.b` in order to be able to express major (ie breaking) version changes in jupyter-fs that are still compatible with jlab 3.x. But this is out of scope for this PR, so let's save the further discussion for a new issue (I'll make one)